doc/asciidoc/consumer/index.adoc

```diff
@@ -151,6 +151,162 @@ include::../producer/data/relationship.created.json[]
 
 the `Schema` strategy leverages the `ids` fields in order to insert/update the relationships so no extra fields will be created.
 
+===== The `Pattern` strategy
```

Comment:
just include it from the connect readme via tagged region

`include::...readme.adoc[tag=pattern-strategy]`

see: https://asciidoctor.org/docs/user-manual/#by-tagged-regions

Reply:
Sorry I realized that I forgot to rename the property names into the `readme.adoc` so we cannot do that